### PR TITLE
Define privacy, Ben's suggestion

### LINKED ISSUE #6: Ben Savage: Need to define Privacy
Note: I am moving gdoc comments into this repo.

Ben Savage writes:

> Should we somehow define what we mean by "privacy" here? If we do not, I can foresee potential future problems where a proposal author might assert that a proposal provides "privacy", but others disagree that it does so - due to a lack of a shared definition of what "privacy" means.

He suggests:
> "No entity should be able to reconstruct a big chunk of their browsing history". I'll try to add a sentence.

### PR DESCRIPTION
**Pull request by martinthomson**
Tweaked some, but essentially as Ben [has proposed](https://github.com/patcg/patwg-charter/pull/20#discussion_r870419713).  Note the addition of
"minimally" here to ensure that we don't foreclose on discussion of
*other* privacy imperatives.  I've also used prose rather than a list,
but ¯\\\_(ツ)\_/¯.

Closes #20.
Closes #6.

### CODE REVIEW

**Comment by AramZS** [comment]
This appears to be what we agreed to on the call. @seanturner ?

File: charter.html [hunk 1]
```diff
@@ -55,7 +55,7 @@
           <li><a href="#background">Background</a></li>
           <li><a href="#scope">Scope</a></li>
           <li><a href="#deliverables">Deliverables</a></li>
-		  <li><a href="#success-criteria">Success Criteria</a></li>
+                  <li><a href="#success-criteria">Success Criteria</a></li>
```
Comment (martinthomson) [inline charter.html:58]:
```suggestion
          <li><a href="#success-criteria">Success Criteria</a></li>
```

File: charter.html [hunk 2]
```diff
@@ -163,8 +163,16 @@ <h2>Scope</h2>
         <p>
             The Working Group will specify new web platform features intended to
             be implemented in browsers or similar user agents. The purpose of
-            these features is to support web advertising and provide users with
-            privacy guarantees with a strong technical basis.
+            these features is to support web advertising without compromising
+            user privacy.  Here "privacy" minimally refers to appropriate
+            process of personal information.  Ways in which new features might
```
Comment (benjaminsavage) [inline charter.html:168]: processing

File: charter.html [hunk 3]
```diff
@@ -163,8 +163,16 @@ <h2>Scope</h2>
...
+            enable inappropriate process include (but are not limited to)
```
Comment (benjaminsavage) [inline charter.html:169]: processing

File: charter.html (hunk 2 above)
Comment (benjaminsavage) [reply charter.html:168]: I think "processing of personal data" is a term of art: https://gdpr-info.eu/art-5-gdpr/

I'm not familiar with "Appropriate process of personal information"

**Review by benjaminsavage — APPROVED** [review]
love it

**Comment by seanturner** [comment]
@AramZS this is the spirit of the changes we discussed on Tuesday. We should have time today to review this.

**Pull request merged**